gslib/commands/version.py

```diff
@@ -169,7 +169,7 @@ def _ComputeCodeChecksum(self):
         m.update(content)
         f.close()
       else:
-        f = open(filepath, 'r')
+        f = open(filepath, 'r', encoding='utf-8')
```

Comment:
At some point once all of this works, we should go back and replace all the `"utf-8"` occurrences with `UTF8` from `gslib.utils.constants`.  There are mixed usages scattered throughout the code already though, so no action needed on this PR :)

Reply:
Changed this one as well as the one below it, since I was doing changes anyway. (hopefully a couple less to change in the future)